src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  // const menAges = (century)
+  //   ? people
+  //     .filter(person => Math.ceil(person.died / 100) === century
+  //       && person.sex === 'm')
+  //   : people
+  //     .filter(person => person.sex === 'm');
+
+  const mens = people.filter(person => person.sex === 'm');
+  const menAges = (century)
+    ? mens.filter(men => Math.ceil(men.died / 100) === century)
+    : mens;
+
+  return menAges
+    .map(person => person.died - person.born)
+    .reduce(getAverage, 0) / menAges.length;
```

Comment:
will be better if you move all this logic to the separate func
```suggestion
  return getAverageAge(menAges)
```